dali/benchmark/displacement_cpu_bench.cc

```diff
@@ -79,19 +79,17 @@ void DisplacementBench(benchmark::State& st) {//NOLINT
 
   // The inputs and outputs to CPUBackend are: shared_ptr<Tensor<CPUBackend>>;
   // create input and output, initialize input
-  std::vector<std::shared_ptr<Tensor<CPUBackend>>> tensor_in;
-  std::vector<std::shared_ptr<Tensor<CPUBackend>>> tensor_out;
-  tensor_in.emplace_back(new Tensor<CPUBackend>());
-  tensor_out.emplace_back(new Tensor<CPUBackend>());
+  auto tensor_in = std::make_shared<TensorVector<CPUBackend>>(1);
```

Comment:
Why not unique_ptr?

Reply:
Cause workspaces operate on shared_ptrs.